### PR TITLE
added test case for consume logic method

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**
fixed review comments.

### CODE REVIEW

File: mock-mv/pom.xml [hunk 1]
```diff
@@ -174,6 +174,40 @@
 			<artifactId>micrometer-registry-prometheus</artifactId>
 			<version>${io.micrometer.prometheus.version}</version>
 		</dependency>
+
+<!--		-->
```
Comment (sowmya695) [inline mock-mv/pom.xml:178]: Remove the commented lines

File: mock-mv/pom.xml [hunk 2]
```diff
@@ -174,6 +174,40 @@
...
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>2.18.3</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito2</artifactId>
```
Comment (sowmya695) [inline mock-mv/pom.xml:195]: If this is not used please remove the depedency

File: mock-mv/src/main/java/io/mosip/mock/mv/controller/MockMvConfigController.java [hunk 3]
```diff
@@ -77,24 +78,26 @@ public ResponseEntity<ConfigureDto> checkConfiguration()
...
+	@RequestMapping(value = "/expectationMockMv", method = RequestMethod.POST)
+	@Operation(summary = "Set expectation", description = "Set expectation", tags = { "Proxy mockMv config API" })
+	@ApiResponses(value = { @ApiResponse(responseCode = "200", description = "OK"),
+			@ApiResponse(responseCode = "201", description = "Created", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(hidden = true))) })
+	public ResponseEntity<String> setExpectation(@Valid @RequestBody Expectation expectation) throws Exception {
+
+		String response = Listener.javaObjectToJsonString(expectation);
```
Comment (sowmya695) [inline mock-mv/src/main/java/io/mosip/mock/mv/controller/MockMvConfigController.java:91]: Change the naming convention

File: mock-mv/src/test/java/TestListner.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
+//package io.mosip.mock.mv;
```
Comment (sowmya695) [inline mock-mv/src/test/java/TestListner.java:1]: Please update package name properly

File: mock-mv/src/test/resources/bootstrap.properties [hunk 5]
```diff
@@ -0,0 +1,5 @@
+spring.cloud.config.uri=http://localhost:51000/config
```
Comment (sowmya695) [inline mock-mv/src/test/resources/bootstrap.properties:1]: Test case should work in offline also. Please remove the url and add properties in application.

**Review by sowmya695 — APPROVED** [review]

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**